### PR TITLE
Nn 2650 cell move non associations

### PR DESCRIPTION
**Pull request by Nimphal**
Screenshot from 2020-08-06 09-47-05

### CODE REVIEW

File: backend/controllers/cellMove/selectLocation.js [hunk 1]
```diff
@@ -51,7 +53,8 @@ module.exports = ({ elite2Api, logError }) => async (req, res) => {
       nonAssociations.nonAssociations.some(
         nonAssociation =>
           nonAssociation.offenderNonAssociation.agencyDescription.toLowerCase() ===
-          prisonerDetails.assignedLivingUnit.agencyName.toLowerCase()
+            prisonerDetails.assignedLivingUnit.agencyName.toLowerCase() &&
```
Comment (JamieDraperUK) [inline backend/controllers/cellMove/selectLocation.js:56]: Sure it's fine but can offenderNonAssociation and assignedLivingUnit be null?

File: backend/controllers/cellMove/viewNonAssociations.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    return res.render('cellMove/nonAssociations.njk', {
+      nonAssociationsRows,
+      breadcrumbPrisonerName: putLastNameFirst(firstName, lastName),
+      prisonerName: formatName(firstName, lastName),
+      dpsUrl,
+      selectLocationLink: `/prisoner/${offenderNo}/cell-move/select-location`,
+    })
+  } catch (error) {
+    if (error) logError(req.originalUrl, error, serviceUnavailableMessage)
```
Comment (JamieDraperUK) [inline backend/controllers/cellMove/viewNonAssociations.js:65]: Can error here sometimes be null? I've seen this code elsewhere and wondered the same thing.

Comment (Nimphal) [reply backend/controllers/cellMove/viewNonAssociations.js:65]: Not if it's an API error I don't think, they follow a pattern.

File: backend/controllers/cellMove/selectLocation.js (hunk 1 above)
Comment (Nimphal) [reply backend/controllers/cellMove/selectLocation.js:56]: Probably :D The API wouldn't return the nonAssociation record without the corresponding offender, and the offender is always serialised the same.

**Pull request merged**